app/frontend/components/pages/sendAda/confirmTransactionDialog.tsx

```diff
@@ -88,34 +88,43 @@ const SendAdaReview = ({
           )}
         </div>
         <div className="ada-label">Amount</div>
-        <div className="review-amount">{printAda(lovelaceAmount as Lovelace)}</div>
+        <div className="review-value">{printAda(lovelaceAmount as Lovelace)}</div>
         {token && formattedAssetItemProps && (
           <FormattedAssetItem {...formattedAssetItemProps}>
-            {({formattedAssetIconName, formattedAmount, formattedFingerprint}) => {
+            {({
+              icon,
+              formattedOnChainName,
+              formattedOffChainName,
+              formattedAmount,
+              formattedFingerprint,
+            }) => {
               return (
                 <Fragment>
-                  <div className="review-label">Token fingerprint</div>
-                  <div className="review-amount">{formattedFingerprint}</div>
-                  <div className="review-label">Token policy Id</div>
-                  <div className="review-amount">{token.policyId}</div>
                   <div className="review-label">Token name</div>
-                  <div className="review-amount">{formattedAssetIconName}</div>
+                  <div className="review-value flex-nowrap">
```

Comment:
Looking at ![this screenshot](https://user-images.githubusercontent.com/69396388/166104200-ca38d497-2d14-4f42-8552-16a975ab37c5.png)

It seems as we were falling back to the onchain name even when showing the `formattedOffChainName` which seems misleading to me. Shouldn't we show just `N/A` here in case there is no offChain name?

Reply:
ok nevermind, I was looking at the wrong side of the picture :)